hedera-node/test-clients/build.gradle.kts

```diff
@@ -226,18 +226,6 @@ tasks.register<Test>("testRepeatable") {
     modularity.inferModulePath.set(false)
 }
 
-tasks.itest {
-    systemProperty("itests", System.getProperty("itests"))
-    systemProperty("junit.jupiter.execution.parallel.enabled", false)
-    systemProperty("TAG", "services-node:" + project.version)
-    systemProperty("networkWorkspaceDir", layout.buildDirectory.dir("network/itest").get().asFile)
-}
-
-tasks.eet {
-    systemProperty("TAG", "services-node:" + project.version)
-    systemProperty("networkWorkspaceDir", layout.buildDirectory.dir("network/itest").get().asFile)
-}
-
```

Comment:
These tasks were not used anymore used in this project. Now they no longer exist. The configuration can be removed.